inst/rmd/h/tocify/jquery.tocify.js

```diff
@@ -385,17 +385,20 @@
         if (this.options.theme !== "bootstrap3") {
 
           item.append($("<a/>", {
-
-            "text": self.text()
+            // Uses html to preserve the text formatting
+            "html": self.html()
 
           }));
 
         } else {
 
-          item.text(self.text());
```

Comment:
Does `item.html(self.html())` work? Having a block element `<p>` inside `<li>` might introduce extra margin/padding issues.

Reply:
Thanks for advice. `item.hmtl(self.html))` works. I've replaced the `<p>` tag.